src/components/Signup/Web2Signup.tsx

```diff
@@ -0,0 +1,239 @@
+			<AuthForm
+				onSubmit={handleSubmitForm}
+				className='flex flex-col gap-y-6'
+			>
+				{
+					isPassword?
+						<>
+							<div className="flex flex-col gap-y-1">
+								<label
+									className="text-base text-sidebarBlue font-medium"
+									htmlFor="first_password"
+								>
+									Set Password
+								</label>
+								<Form.Item
+									name="first_password"
+									rules={[
+										{
+											message: messages.VALIDATION_PASSWORD_ERROR,
+											required: password.required
+										},
+										{
+											message: messages.VALIDATION_PASSWORD_ERROR,
+											min: password.minLength
+										}
+									]}
+								>
+									<Input.Password
+										onChange={(e) => {
+											setFirstPassword(e.target.value);
+										}}
+										placeholder='Password'
+										className="rounded-md py-3 px-4"
+										id="first_password"
+									/>
+								</Form.Item>
+							</div>
+							<div className="flex flex-col gap-y-1">
+								<label
+									className="text-base text-sidebarBlue font-medium"
+									htmlFor="second_password"
+								>
+									Re-enter Password
+								</label>
+								<Form.Item
+									name="second_password"
+									rules={[
+										{
+											message: 'Password don\'t match',
+											validator(rule, value, callback) {
+												if (callback && value !== firstPassword){
+													callback(rule?.message?.toString());
+												}else {
+													callback();
+												}
+											}
+										}
+									]}
+								>
+									<Input.Password
+										placeholder='Password'
+										className="rounded-md py-3 px-4"
+										id="second_password"
+									/>
+								</Form.Item>
+							</div>
+						</>
+						:<>
+							<div className="flex flex-col gap-y-1">
+								<label
+									className="text-base text-sidebarBlue font-medium"
+									htmlFor="username"
+								>
+                        Username
+								</label>
+								<Form.Item
+									name="username"
+									rules={[
+										{
+											message: messages.VALIDATION_USERNAME_REQUIRED_ERROR,
+											required: username.required
+										},
+										{
+											message: messages.VALIDATION_USERNAME_PATTERN_ERROR,
+											pattern: username.pattern
+										},
+										{
+											max: username.maxLength,
+											message: messages.VALIDATION_USERNAME_MAXLENGTH_ERROR
+										},
+										{
+											message: messages.VALIDATION_USERNAME_MINLENGTH_ERROR,
+											min: username.minLength
+										}
+									]}
+								>
+									<Input
+										placeholder="John"
+										className="rounded-md py-3 px-4"
+										id="username"
+									/>
+								</Form.Item>
+							</div>
+							<div className="flex flex-col gap-y-1">
+								<label
+									htmlFor="email"
+									className="text-base text-sidebarBlue font-medium"
+								>
+						Email
```

Comment:
Please add the helper tooltip added in design, and the spacing between the email label and the previous input seems not to be as per the design (48px)